src/main/java/entities/User.java

```diff
@@ -0,0 +1,133 @@
+package entities;
+
+public class User {
+    // ArrayList of User's favourited posts
+    private ArrayList<Post> favourites;
+
+    // ArrayList of Posts User has made
+    private ArrayList<Post> posts;
+
+    // boolean relating if User has admin privileges
+    private boolean isAdmin;
+
+    // Number of Users created
+    public static int numUsers = 0;
+
+    // id is the unique identifier of User, equal to number of posts at time of creation
+    private int id;
+
```

Comment:
attributes for email, password

Reply:
Adding these attributes, and adding them to constructor and getter methods.